### ISSUE TITLE
SimpleBar doesn't work in latest version

### ISSUE THREAD

**Issue by NhatDuy1407** [Grsmto/simplebar#223]
Hi @Grsmto 
Could you please double check the latest code? And fix it.

**Comment by Relaxable** [comment]
Has there been a change in the file located at https://unpkg.com/simplebar@latest/dist/simplebar.css as my design seems to be a bit broken?

**Comment by Grsmto** [comment]
Hi @NhatDuy1407 could you test again? I did another release.

**Comment by Grsmto** [comment]
@Relaxable yes if you use the `@latest` tag you will always get the latest version so things can break when a breaking change is released!
You should fix it to a version instead, like `https://unpkg.com/simplebar@3.0.0-beta.5/dist/simplebar.css` for the previous version.

**Comment by Relaxable** [comment]
> 
> 
> @Relaxable yes if you use the `@latest` tag you will always get the latest version so things can break when a breaking change is released!
> You should fix it to a version instead, like `https://unpkg.com/simplebar@3.0.0-beta.5/dist/simplebar.css` for the previous version.

Thanks, didn't even check the version I was using. I see now there has been a recent release.

**Comment by adjourn** [comment]
@Grsmto New versions don't include anything from `next`, is it safe to start playing with `next` yet? Really need those sticky elements to work in my dev phase.

**Comment by MazenTouati** [comment]
I confirm too. The latest version seems to cause the double scroll bar issue. The scroll bar produced by the plugin will scroll the content just fine. but the native scroll bar will scroll the parent and shows only white space, because the content masked by the mask's div i think.

**Comment by Grsmto** [comment]
Could you guys please test the latest version? I've been working on the release process recently as it was quite messed up. Should be more streamlined for now.
If you still experience issues, could you please provide a jsfiddle with your issue? That would help **a lot!**

**Comment by flofleche** [comment]
Hi @Grsmto, thanks for your nice work ! 
I'm currently using your plugin in a new project and it really makes my UI looking better and yet light.

Though I realized while trying to update from the `3.0.0-beta.5` to the latest `3.1.0-beta.5` that there was few things that doesn't work quite well for me (my wander is whether it could be coming from my own template or whether it is due to some bugs inside the plugin..?).

Here is few things that I have noticed :
- `autoHide`'s option doesn't work the way it was (compare to `3.0.0-beta.5`). Scrollbar now disappears right after a single interaction with it.
- Although, I've try to change classes through the option `classNames` but then most of them happen to be set to `class="undefined"` when looking into the DOM.. 
- Otherwise something tells me that the documentation may not be yet fully updated ??
What about the `forceVisible` option ? Is something that can be used already ?..

Anyway there might be still a bunch of issues that you are probably trying to solve and I don't mean to sounds pushy about it, so I'll keep looking around for new updates when they might become available.

Thanks one more time for all the great work you are giving on this project !
